### PR TITLE
Add version numbers to featureset search

### LINKED ISSUE #246: Add version numbers to featureset search dialog

### PR DESCRIPTION
**Pull request by msmolens**
Fixes #246

### CODE REVIEW

**Comment by brianhelba** [comment]
One thing I realized is that `girder.Model` has [a `name()` function](https://github.com/girder/girder/blob/ae461ee437b9a788cd2ed2d2e47b567ac0996de8/clients/web/src/model.js#L28), which defaults to `this.get('name');`, but could be overridden by subclasses. Perhaps we should just be more consistent about only using `Model.name` to get rendered names, then override that function in `isic.models.FeaturesetModel`.

@msmolens What do you think?

**Comment by msmolens** [comment]
@brianhelba I think at least for `FeaturesetModel` it makes sense to override `name()`. I will update this PR accordingly.

**Comment by brianhelba** [comment]
@msmolens Per our discussion, I'll work on this.

**Comment by brianhelba** [comment]
@msmolens PTAL. This goes a little beyond the scope of the original issue, but I think it will be more maintainable to use `.name()` to render our JS generally.

File: web_external/js/views/body/CreateStudyView.js [hunk 1]
```diff
@@ -46,7 +46,7 @@ isic.views.CreateStudyView = isic.View.extend({
             getInfoCallback: function (type, obj) {
                 if (type === 'featureset.isic_archive') {
                     return {
-                        text: obj.name,
+                        text: (new isic.models.FeaturesetModel(obj)).name(),
```
Comment (msmolens) [inline web_external/js/views/body/CreateStudyView.js]: I'd prefer to assign the new object to a named variable to simplify the syntax, but I'm ok if we don't change it too.

**Comment by msmolens** [comment]
@brianhelba Nice, much cleaner! Please take a look at the commits I added.

**Pull request merged**